corehq/apps/translations/models.py

```diff
@@ -92,3 +100,45 @@ def get_translations(cls, lang, key=None, one=False):
                 return dict([(key, val[0]) for key, val in translations.items()])
             else:
                 return translations
+
+
+FIELD_NAME_HELP = """
+Usually the string in either case list or detail under 'property'.
+This could be an xpath or case property name.
+If it is an ID Mapping then the property should be '<property> (ID Mapping Text)'.
+For the values each value should be '<id mapping value> (ID Mapping Value)'.
+Example: case detail for tasks_type would have entries:
+    tasks_type (ID Mapping Text)
+    child (ID Mapping Value)
+    pregnancy (ID Mapping Value)
+"""
+
+
+class TransifexBlacklist(models.Model):
+    """Used for removing case list and case detail translations before an
+    upload to Transifex.
+
+    This assumes that a default source translation is English exists.
+
+    Note that field_name is not sufficient to exclude properties as you can
+    have two details in the same module that display the same information in a
+    different way e.g. date of birth and age in years. display_text is used to
+    determine which trnaslations to hold back from Transifex
+    """
+    domain = models.CharField(max_length=255)
+    app_id = models.CharField(max_length=32)
+    module_id = models.CharField(max_length=32)
+    field_type = models.CharField(
+        max_length=100,
+        choices=(
+            ('detail', 'Case Detail'),
+            ('list', 'Case List'),
+        )
+    )
+    field_name = models.TextField(help_text=FIELD_NAME_HELP)
+    display_text = models.TextField(
+        help_text="The default language's translation for this detail/list. "
+        "If display_text is not filled out then all translations will be blacklisted")
```

Comment:
`all` can be misinterpreted here as all translations. You can say all translations matching field_type and field_name instead